rs/execution_environment/src/hypervisor/tests.rs

```diff
@@ -7848,3 +7848,36 @@ fn ic0_mint_cycles_u64() {
             >= 2 * (1 << 64) - 10_000_000
     );
 }
+
+fn check_correct_execution_state(is_wasm64: bool) {
+    let mut test = if is_wasm64 {
+        ExecutionTestBuilder::new().with_wasm64().build()
+    } else {
+        ExecutionTestBuilder::new().build()
+    };
```

Comment:
I think we want the feature enabled even when testing the 32-bit memory, right? We want to show we properly detect 32-bit Wasms when the feature is enabled.

Reply:
good catch, thanks!